src/Kunstmaan/AdminBundle/Resources/ui/js/_input-range-slider.js

```diff
@@ -0,0 +1,53 @@
+var kunstmaanbundles = kunstmaanbundles || {};
+
+kunstmaanbundles.rangeslider = (function(window, undefined) {
+
+    var init, reInit, initRangePercentage;
+    var el, newPoint, newPlace, offset;
+
+    init = reInit = function() {
+        $(".range").find('input').change(function() {
```

Comment:
Check if the element isn't already initialised? (cfr. the characters-left.js file)